bazel/protobuf.bzl

```diff
@@ -18,21 +18,24 @@ load("@rules_proto//proto:defs.bzl", "ProtoInfo")
 _PROTO_EXTENSION = ".proto"
 _VIRTUAL_IMPORTS = "/_virtual_imports/"
 
+_WELL_KNOWN_PROTOS_BASE = [
+	"any_proto",
+	"api_proto",
+	"compiler_plugin_proto",
+	"descriptor_proto",
+	"duration_proto",
+	"empty_proto",
+	"field_mask_proto",
+	"source_context_proto",
+	"struct_proto",
+	"timestamp_proto",
+	"type_proto",
+	"wrappers_proto",
+]
+
 def well_known_proto_libs():
-    return [
-        "@com_google_protobuf//:any_proto",
-        "@com_google_protobuf//:api_proto",
-        "@com_google_protobuf//:compiler_plugin_proto",
-        "@com_google_protobuf//:descriptor_proto",
-        "@com_google_protobuf//:duration_proto",
-        "@com_google_protobuf//:empty_proto",
-        "@com_google_protobuf//:field_mask_proto",
-        "@com_google_protobuf//:source_context_proto",
-        "@com_google_protobuf//:struct_proto",
-        "@com_google_protobuf//:timestamp_proto",
-        "@com_google_protobuf//:type_proto",
-        "@com_google_protobuf//:wrappers_proto",
-    ]
+    return (["@com_google_protobuf//:" + b for b in _WELL_KNOWN_PROTOS_BASE] +
+                ["@com_google_protobuf//src/google/protobuf:" + b for b in _WELL_KNOWN_PROTOS_BASE])
```

Comment:
Will we always need both of them in the future?

Reply:
Added a comment explaining why both are present here.